src/components/FocusTrap/FocusTrapForScreen/index.web.tsx

```diff
@@ -0,0 +1,67 @@
+import {useFocusEffect, useIsFocused, useRoute} from '@react-navigation/native';
+import FocusTrapOriginal from 'focus-trap-react';
+import React, {useCallback, useMemo} from 'react';
+import BOTTOM_TAB_SCREENS from '@components/FocusTrap/BOTTOM_TAB_SCREENS';
+import SCREENS_WITH_AUTOFOCUS from '@components/FocusTrap/SCREENS_WITH_AUTOFOCUS';
+import sharedTrapStack from '@components/FocusTrap/sharedTrapStack';
+import WIDE_LAYOUT_INACTIVE_SCREENS from '@components/FocusTrap/WIDE_LAYOUT_INACTIVE_SCREENS';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import type FocusTrapProps from './FocusTrapProps';
+
+let activeRouteName = '';
```

Comment:
Do we need this outside the component? Can't we just use, for example, useRef?

Reply:
This variable is shared between instances of `FocusTrap` component